templates/index.html

```diff
@@ -138,6 +140,17 @@
         </paper-toolbar>
         <div id="container" class="container" layout="" vertical="" center="">
           <x-schedule id="mainschedule" show="all"></x-schedule>
+          <div id="GithubUpdateBox" style="text-align:center; background-color:white;">
+            <p style="padding:12px; margin-bottom:12px; margin-top:0px; font-size:1.4rem; font-weight:300;">
```

Comment:
I'd prefer if this CSS was in the .css file but we can do that in a future PR if preferred